tests/e2e/tests/mixer/mixer_test.go

```diff
@@ -589,6 +594,7 @@ func TestRateLimit(t *testing.T) {
 		errorf(t, "Bad metric value for successful requests (200s): got %f, want at least %f", got, want)
 	}
 
+	want200s = math.Ceil(want200s * 1.1) // timing is short, allow 10% extra for rounding errors and quota bucket size
```

Comment:
down this path is madness ;). this test has always had fuzzy boundaries though, so this should be fine.